common/djangoapps/course_modes/tests/test_views.py

```diff
@@ -43,44 +44,58 @@ class CourseModeViewTest(CatalogIntegrationMixin, UrlResetMixin, ModuleStoreTest
     @patch.dict(settings.FEATURES, {'MODE_CREATION_FOR_TESTING': True})
     def setUp(self):
         super(CourseModeViewTest, self).setUp()
-        self.course = CourseFactory.create()
+        now = datetime.now(pytz.utc)
```

Comment:
Please add a comment on this time setup work

Reply:
added comment below and in the method where the variables are used